tests/test_physical_channel_property.py

```diff
@@ -0,0 +1,110 @@
+"Contains a collection of pytest tests that validates the physical channel properties."
+import numpy
+import pytest
+
+from nidaqmx.constants import TerminalConfiguration
+
+
+def test__physical_channel__get_bool_property__returns_value(any_x_series_device):
+    """Test for validating boolean attributes in physical channel."""
+    phys_chans = any_x_series_device.di_lines
+    assert phys_chans[0].di_change_detect_supported
+
+
+def test__physical_channel_with_teds__get_bit_stream__returns_configured_value(
+    any_x_series_device, teds_file_path
+):
+    """Test for validating byte attributes in physical channel."""
+    phys_chans = any_x_series_device.ai_physical_chans
+    expected_value = numpy.array(VALUES_IN_TED, dtype=numpy.uint8)
+
+    # Generate path to a virtual TEDS file.
+    phys_chans["ai0"].configure_teds(teds_file_path)
```

Comment:
TEDS is part of the given clause ("physical_channel_with_teds"), so `configure_teds` should be part of the arrange section. Remove the blank line between `expected_value = ...` and `configure_teds`.

Same for all of these TEDS test cases.